tools/libboard/board_dc/board_dc.c

```diff
@@ -326,3 +340,126 @@ fpga_result print_board_info(fpga_token token)
 
 	return resval;
 }
+
+// Read sysfs
+fpga_result read_sysfs(fpga_token token, char *sysfs_path, char *sysfs_name)
```

Comment:
When passing strings around, we should also pass the buffer size as a size_t.  Otherwise, we cannot know how large the storage behind these buffers actually is.

Reply:
moved common code libborad_common.so file

Reply:
fixed